addons/compat_spe/config.cpp

```diff
@@ -0,0 +1,53 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+            "SPE_M4A1_75_Command",
+            "SPE_M4A1_75_Command_DVL",
+            "SPE_FR_M4A1_75_Command",
+            "SPE_FR_M4A1_75_Command_DVL",
+            "SPE_M4A1_76_Command",
+            "SPE_M4A1_76_Command_DVL",
+            "SPE_FR_M4A1_76_Command",
+            "SPE_FR_M4A1_76_Command_DVL",
+            "SPE_M4A0_75_Early_Command",
+            "SPE_M4A0_75_Early_Command_DVL",
+            "SPE_M4A0_75_Command",
+            "SPE_M4A0_75_Command_DVL",
+            "SPE_FR_M4A0_75_Early_Command",
+            "SPE_FR_M4A0_75_Early_Command_DVL",
+            "SPE_FR_M4A0_75_mid_Command",
+            "SPE_FR_M4A0_75_mid_Command_DVL",
+            "SPE_PzBefWgIII_K",
+            "SPE_PzBefWgIII_K_DLV",
+            "SPE_ST_PzBefWgIII_K",
+            "SPE_ST_PzBefWgIII_K_DLV",
+            "SPE_PzBefWgIV",
+            "SPE_PzBefWgIV_DLV",
+            "SPE_ST_PzBefWgIV",
+            "SPE_ST_PzBefWgIV_DLV",
+            "SPE_PzBefWgVI",
+            "SPE_PzBefWgVI_DLV",
+            "SPE_ST_PzBefWgVI",
+            "SPE_ST_PzBefWgVI_DLV"
```

Comment:
Are those new classes or why are they here? Modified classes don't need to be in `units[]`, only where they are added.

Reply:
Some are ours, some are in SPE but scope=1 and possibly not in their `units[]`

Reply:
I am a bit confused then. Why are we adding `SPE_`-prefixed classes in a compatibility? :thinking:

Reply:
This is how the SPE team wanted me to do it. Many classnames do already exist in SPE and I think the rest will end up there eventually, just wasn't time to do it before their release.

Reply:
Ah I see. Will any inheritance change and break things, so we should re-add later, or was this planned out and we can go ahead?

Reply:
Planned out and can go ahead. SPE has a cooy in their own repo.